src/components/Box.js

```diff
@@ -1,8 +1,14 @@
+import Later from './Later';
+import Joker from '../media/images/joker.jpg';
+//import Arrow from '../media/images/arrow.png';
+
 const Box = (props) => {
```

Comment:
Możesz tutaj uzyć destrukturyzacji
```suggestion
const Box = ({ link, name }) => {
```